src/pages/ItemList.js

```diff
@@ -118,7 +117,18 @@ function ItemList({ token }) {
     return filteredData;
   };
 
-  const list = filterList();
+  // eslint-disable-next-line array-callback-return
```

Comment:
Ideally we shouldn't need to disable es-lint rules because they're usually telling us something. In this case, I believe that it's that we should have a something return in none of the conditional statements evaluate to true. I think that if we do something like `return 0;` at the end, then we won't need to disable the linter rule.

Reply:
Will do!